yaml_lib/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Library to handle all the YAML related functions"""
+
+from yaml_lib.load_yaml import load_yaml_from_path
```

Comment:
I think you could also write, but it really don't change anything
```suggestion
from .load_yaml import load_yaml_from_path
```